src/app/containers/CpsRecommendations/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,195 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CpsRecommendations should not render when cpsRecommendations toggle is disabled 1`] = `<div />`;
+
+exports[`CpsRecommendations should not render when the hasStoryRecommendations flag is disabled for the service 1`] = `<div />`;
+
+exports[`CpsRecommendations should render for a single recommendation item 1`] = `
+<div>
+  <div
+    class="WsojWrapper-sc-1p91yk0-3 bybDJb"
+  >
+    <section
+      aria-labelledby="recommendations-heading"
+      class="Wrapper-opyob2-0 dlivvF"
+      data-e2e="related-content"
+      role="region"
+    >
+      <div
+        class="SectionLabelWrapper-sc-1p91yk0-2 keYZPs"
+      >
+        <strong
+          aria-labelledby="recommendations-heading"
+          class="CustomSectionLabel-sc-1p91yk0-1 dbbYpP"
+          dir="ltr"
+        >
+          Recommendations
+        </strong>
+      </div>
+      <div
+        class="SingleContentWrapper-opyob2-3 iddZTf"
+      >
+        <div
+          class="GridComponent-nf79gm-0 elxESl"
+          dir="ltr"
+        >
+          <div
+            class="StoryPromoWrapper-sc-1dvfmi3-0 buBJys StoryPromoWrapper-sc-1p91yk0-0 gsqqaw"
+          >
+            <div
+              class="ImageGridItem-sc-1xk56cj-0 fNYKlW"
+              dir="ltr"
+            >
+              <div
+                class="ImageContentsWrapper-sc-1dvfmi3-1 cinwNl"
+              >
+                <img
+                  alt=""
```

Comment:
missing a value for the alt attribute?

Reply:
confirmed from @lizcameron that we currently don't have a value for the alt attribute but it's on our list of improvements, using a placeholder value for now until the value becomes available